fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -145,6417 +145,6708 @@
 
 @Entity
```

Comment:
Note to self: in this mammoth file there are two effective changes : getter setter for loan type and GLIM object.